src/components/forms/validations.jsx

```diff
@@ -23,6 +23,7 @@ module.exports.validations = {
         return phoneNumberUtil.isValidNumber(parsed);
     }
 };
+module.exports.validations.notEqualsUsername = module.exports.validations.notEquals;
```

Comment:
We can't have different error messages for the same validation on one field, yeah.